services/MerchStoreService.ts

```diff
@@ -279,6 +310,72 @@ export default class MerchStoreService {
     });
   }
 
+  /**
+   * TODO: add global variable
+   * Verify that items have valid options. An item with variants disabled cannot have multiple
+   * options, and an item with variants enabled cannot have multiple option types.
+   */
+  private static verifyItemHasValidPhotos(item: MerchItem | MerchandiseItemModel) {
+    if (item.photos.length > 5) {
+      throw new UserError('Merch items cannot have more than 5 pictures');
+    }
+  }
+
+  /**
+   * Creates an item photo and assign it the corresponding picture url
+   * and append the photo to the photos list from merchItem
+   * @param item merch item uuid
+   * @param properties merch item photo picture url and position
+   * @returns created item photo
+   */
+  public async createItemPhoto(item: Uuid, properties: MerchItemPhoto): Promise<PublicMerchItemPhoto> {
+    return this.transactions.readWrite(async (txn) => {
+      const merchItem = await Repositories.merchStoreItem(txn).findByUuid(item);
+      if (!merchItem) throw new NotFoundError('Merch item not found');
+
+      // add to the end
+      const position = merchItem.photos.length;
+
```

Comment:
Per my comment above on positioning, let's not have any logic around updating positions. We should just make sure that it's a field passed in to `properties` above so that the frontend can pass us the position and we just store it internally

Reply:
added